internal/olm/operator/bundle/install.go

```diff
@@ -51,6 +51,7 @@ func (i *Install) BindFlags(fs *pflag.FlagSet) {
 
 	// --mode is hidden so only users who know what they're doing can alter add mode.
 	fs.StringVar((*string)(&i.BundleAddMode), "mode", "", "mode to use for adding bundle to index")
+	fs.BoolVar(&i.IndexImageCatalogCreator.AllowInsecure, "allow-insecure", false, "allows pulling bundle image from an insecure registry server")
```

Comment:
How about mirroring `opm` and naming this
```suggestion
	fs.BoolVar(&i.IndexImageCatalogCreator.SkipTLS, "skip-tls", false, "skip authentication of image registry TLS certificate when pulling a bundle image in-cluster")
```

Reply:
> How about mirroring `opm` and naming this

Sounds good, i will update the PR.

Reply:
Renamed the option as suggested. Also moved the flag to `IndexImageCatalogCreator` in `internal/olm/operator/registry/index_image.go` so the option will be available to both `run bundle` and `run bundle-upgrade`.